third_party/eth2_types.BUILD

```diff
@@ -0,0 +1,39 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "bls.go",
+        "blsprivatekey.go",
+        "blspublickey.go",
+        "blssignature.go",
+        "domain.go",
+        "errors.go",
+        "privatekey.go",
+        "publickey.go",
+        "signature.go",
+    ],
+    importpath = "github.com/wealdtech/go-eth2-types/v2",
+    importpath_aliases = ["github.com/wealdtech/go-eth2-types"],
+    visibility = ["//visibility:public"],
+    deps = [
+        "@herumi_bls_eth_go_binary//:go_default_library",
```

Comment:
We're going to want to remove this dependency somehow. We're still working on revamp design for integrating the functionality of wallet / keymanager, but I am thinking we can agree on some interface that would be implemented by `//shared/bls:go_default_library`.

Reply:
No problems with doing this once it's available.  May be worth raising as a separate issue, rather than holding back this patch for it, given that any users on the current testnet who are using the `wallet` keymanager won't be able to sign correctly without this patch in place and the testnet goes live in around 24 hours.

Reply:
Agreed on that. No delay for this PR. Thanks!